tools/asciidoctor/cli.adoc

```diff
@@ -0,0 +1,197 @@
+// Copyright (C) 2020  Matthew Glazar
+// See end of file for extended copyright information.
+
+= quick-lint-js(1)
+:doctype: manpage
+:version: 0.2.0
+:manmanual: General Commands Manual
```

Comment:
Nit: Is this required? I don't like "General Commands Manual" at the top of the man page personally. =S

Reply:
Same, I was just copying the original man page.

Reply:
> Same, I was just copying the original man page.

Is that in docs/quick-lint-js.1 already? On my macOS machine, there is no title. Perhaps Linux' `man` toolchain adds it automatically?

Reply:
It probably is Linux behavior. If I delete the manmanual attribute, it gets rid of General Commands Manual on Linux.

```diff
-.\"    Manual: General Commands Manual
+.\"    Manual: \ \&
 .\"    Source: quick-lint-js version 0.2.0
 .\"  Language: English
 .\"
-.TH "QUICK\-LINT\-JS" "1" "" "quick\-lint\-js version 0.2.0" "General Commands Manual"
+.TH "QUICK\-LINT\-JS" "1" "" "quick\-lint\-js version 0.2.0" "\ \&"
```